--- /usr/src/jcomeauictx/aports/main/tk/src/tk8.6.10/unix/tkUnixRFont.c.orig	2025-10-16 07:45:41.682704846 -0700
+++ /usr/src/jcomeauictx/aports/main/tk/src/tk8.6.10/unix/tkUnixRFont.c	2025-10-16 08:04:53.444127590 -0700
@@ -84,7 +84,8 @@
     double angle)
 {
     int i;
-
+    UnixFtFont *result;
+    fprintf(stderr, "GetFont(%d, %0.3f) called\n", ucs4, angle);
     if (ucs4) {
 	for (i = 0; i < fontPtr->nfaces; i++) {
 	    FcCharSet *charset = fontPtr->faces[i].charset;
@@ -153,7 +154,9 @@
 	    fontPtr->faces[i].angle = angle;
 	}
     }
-    return (angle==0.0? fontPtr->faces[i].ft0Font : fontPtr->faces[i].ftFont);
+    result = (angle==0.0? fontPtr->faces[i].ft0Font : fontPtr->faces[i].ftFont);
+    fprintf(stderr, "GetFont returning \"%s\"\n", result);
+    return result;
 }
 
 /*
